app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsPresenter.kt

```diff
@@ -7,4 +7,12 @@ class NewsPresenter @Inject constructor() : BasePresenter<INewsView>() {
 
     override fun onViewAttached() {
     }
+
+    fun addNewsPressed() {
```

Comment:
You could make it one-line function like `fun addNewsPressed() = view.goAddNews()` and get a cleaner code.